main.swift

```diff
@@ -65,6 +65,7 @@ class StringReplacer {
     private let authorEmail: String
     private let gitHubURL: String
     private let year: String
+    private let today: String
```

Comment:
How about we call this `dateString` or something instead? I find it a bit more descriptive than `today` 🙂